src/patches/Node.js

```diff
@@ -79,7 +79,11 @@ function scheduleObserver(node, addedNode, removedNode) {
   const observer = nodeData && nodeData.observer;
   if (observer) {
     if (addedNode) {
-      observer.addedNodes.push(addedNode);
+      if (addedNode.nodeType === Node.DOCUMENT_FRAGMENT_NODE) {
+        observer.addedNodes.push(...addedNode.childNodes);
```

Comment:
Do you want me to use `childNodesArray` here?

Reply:
Think this is fine.